pkg/legacy/converter_test.go

```diff
@@ -48,32 +48,42 @@ func TestBuildProxySettings(t *testing.T) {
 
 	value, err = buildProxySettings(agentConfig)
 	assert.Nil(t, err)
-	assert.Equal(t, "http://foobar.baz", value)
+	proxyOnlyHost := make(map[string]string)
```

Comment:
You can initialize it at creation (and test both `http` and `https`, following my previous comment):
```go
proxyOnlyHost := map[string]string{
    "http": "http://foobar.baz",
    "https": "http://foobar.baz",
}
```